src/test/java/org/folio/rest/impl/PurchaseOrdersApiTest.java

```diff
@@ -2415,6 +2417,112 @@ public void testPutOrderstoConvertToIsbn13() throws Exception {
     assertThat(MockServer.getPoLineUpdates().get(0).mapTo(PoLine.class).getDetails().getProductIds().get(0).getProductId(), equalTo("9780198526636"));
   }
 
+
+  @Test
+  public void testPostOrderWithUserNotHavingApprovalPermissions() {
+    logger.info("===  Test case when approval is required and user does not have required permission===");
+
+    CompositePurchaseOrder reqData = getMockAsJson(COMP_ORDER_MOCK_DATA_PATH, PO_ID_OPEN_TO_BE_CLOSED).mapTo(CompositePurchaseOrder.class);
+    reqData.setVendor("d0fb5aa0-cdf1-11e8-a8d5-f2801f1b9fd1");
```

Comment:
I realize this is already used throughout the file, but maybe it makes sense to define a static constant instead of repeating the literal over and over again.

Reply:
changed to use constant